message.mk

```
@@ -84,8 +84,7 @@ MSG_FILE_TOO_SMALL = The firmware is too small! $(CURRENT_SIZE)/$(MAX_SIZE)\n
 MSG_FILE_JUST_RIGHT = The firmware size is fine - $(CURRENT_SIZE)/$(MAX_SIZE) ($(PERCENT_SIZE)%%, $(FREE_SIZE) bytes free)\n
 MSG_FILE_NEAR_LIMIT = The firmware size is approaching the maximum - $(CURRENT_SIZE)/$(MAX_SIZE) ($(PERCENT_SIZE)%%, $(FREE_SIZE) bytes free)\n
 MSG_PYTHON_MISSING = $(WARN_COLOR)WARNING:$(NO_COLOR)\n \
-	Python 3 is not installed. It will be required by a future version\n\
-	of qmk_firmware.\n\n\
+	Can not run bin/qmk! This will be required after 2020 Aug 30.\n\n\
```

Do we not want to align the message here with the one in the makefile?
```suggestion
	Can not run bin/qmk! This will be a failing check after the next future merge.\n\n\
```

---

Probably should be "after the next merge of the `develop` branch", assuming we're doing that pivot after current breaking changes cycle.

---

I think we should put a date in there. I'd be up for tweaking the wording though.

```suggestion
	Can not run bin/qmk! This tool will be required when the develop branch is merged on 2020 Aug 30.\n\n\
```